lib/drivers/insulet/insuletDriver.js

```diff
@@ -652,10 +652,12 @@ module.exports = function (config) {
         continue;
       }
       if (rec.error_code) {
-        debug('Nonzero logRecord error_code!');
+        // according to the spec, a record with an error code should not be parsed
```

Comment:
This was actually on the checklist from our meeting. Maybe the checklist should be updated?